src/vtok_p11/src/crypto/mod.rs

```diff
@@ -232,7 +232,7 @@ fn bignum_to_vec(bn: *const ffi::BIGNUM) -> Result<Vec<u8>> {
     }
     let mut ret = vec![0u8; len as usize];
 
-    let written = unsafe { ffi::BN_bn2bin(bn, ret.as_mut_ptr() as *mut i8) };
+    let written = unsafe { ffi::BN_bn2bin(bn, ret.as_mut_ptr() as *mut _) };
     if ret.len() != written as usize {
```

Comment:
Since this does an inferred cast, maybe it makes sense to replace as usize with try_into() here, such that it doesn't overflow.

Reply:
True, but the return type from the aws-lc FFI for this is `size_t`, which along `usize` both have `u64` representation.